utils/debufr.f

```diff
@@ -0,0 +1,461 @@
+C> @file
```

Comment:
why C and fortran both?

Reply:
This utility has a lot of UNIX command line options, which at least historically are generally easier and more standard to process in C than in Fortran.  So the driver is written in C, and then most of the remaining code logic is in Fortran.  The library itself has a similar mix of C and Fortran, and as long as the interface across the languages is clean and deliberate, it works and is portable across all platforms.